Rocket.Chat/Extensions/UINavigationBar/UINavigationItemAccessibilityExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UINavigationItemAccessibilityExtension.swift
+//  Rocket.Chat
+//
+//  Created by Rudrank Riyam on 29/03/19.
+//  Copyright © 2019 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationItem {
+    func applyMoreButtonAccessibility() {
```

Comment:
Same concerns as for the `UIButton` thing, but since you are working with the nav bar item for the authentication controllers, I think you should work with the `AuthNavigationController`. That should allow you to do all your setup in one place.

Reply:
Yes, creating an extension just for the accessibility is not a good idea. I will move the methods into the AuthNavigationController. 
Thank you!